sqlmesh/core/model/definition.py

```diff
@@ -1658,6 +1682,7 @@ def _create_model(
                 "jinja_macros": jinja_macros,
                 "dialect": dialect,
                 "depends_on": depends_on,
+                "physical_schema_override": physical_schema_override.get(exp.to_table(name).db),
```

Comment:
I'm a bit confused - do we or do we **not** have to pass dialect into something like `exp.to_table`, @tobymao ?

Reply:
does the name go through the normalizer? if not then this is incorrect. you need to make sure the the model name validator runs and use that and then you can properly get the db